RELEASING.md

```diff
@@ -9,30 +9,36 @@ bundle exec pod trunk info BlueprintUI
 bundle exec pod trunk info BlueprintUICommonControls
 ```
 
-See [the CocoaPods documentation for pod trunk](https://guides.cocoapods.org/making/getting-setup-with-trunk) for more information about setting up credentials on your device.
+See [the CocoaPods documentation for pod trunk](https://guides.cocoapods.org/making/getting-setup-with-trunk) for more information about setting up credentials on your device. If you need to be added as an owner, ping in #blueprint on Slack (Square only).
 
-1. Make sure you're on the `master` branch
+1. Make sure you're on the `master` branch, and `git pull` to get the latest commits.
 
-1. Make sure that the library version in both `BlueprintUI.podspec` and `BlueprintUICommonControls.podspec` is correct (it should match the version number that you are about to release).
+1. Create a branch off `master` to update the version numbers and `Podfile.lock`. An example name would be `kve/release-0.9`.
+
+1. Update the library version in both `BlueprintUI.podspec` and `BlueprintUICommonControls.podspec` if it has not already been updated (it should match the version number that you are about to release).
 
 1. Run `bundle exec pod install` to update the `Podfile.lock` with the new versions.
```

Comment:
Can you add "from the SampleApp directory"? Been meaning to change this since it moved from root.

Reply:
+1

Reply:
Actually, thought: Should we just un-nest the podfile? Always found it weird that it was in `SampleApp`, given it's a critical part of how we locally consume and  publish the `podspecs`, which are at the root.

Reply:
I like the setup that @amorde set up for Workflow with a Development podspec that contains all the samples: https://github.com/square/workflow/pull/964 (I don’t _think_ we’d even need the Podfile in that case)

Reply:
Either way is fine with me. The Podfile isn't related to publishing, it's only used by the SampleApp.

Reply:
I don't think the podspec-only tricks without a Podfile because you have no way to point to other local pods, right? Eg, in that example, doesn't this:

```
  s.dependency 'Workflow'
  s.dependency 'WorkflowUI'
```

Use the published versions of Workflow because it doesnt know to look locally? (Or maybe that's been resolved?)

Reply:
You can point to local podspecs if necessary either using a Podfile or (what we did) using the `--local-sources` CLI option https://github.com/square/workflow/pull/964/files#diff-bc5b13d6a0956bdb7636d99b59384e18R1-R4

Reply:
Huh, interesting, TIL!